cypress/e2e/shared/telegrafs.test.ts

```diff
@@ -76,7 +76,16 @@ describe('Collectors', () => {
           const buckets = bucketz.slice(0).sort((a, b) => {
             const _a = a.toLowerCase()
             const _b = b.toLowerCase()
-            return _a > _b ? 1 : _a < _b ? -1 : 0
+
+            if (_a > _b) {
+              return 1
+            }
+
+            if (_a < _b) {
+              return -1
+            }
+
+            return 0
```

Comment:
while this is a correct undoing of the ternary operator, why not use the inbuilt string localeCompare?  (it's optimized so should be faster)

Reply:
>why not use the inbuilt string localeCompare?

@jrenee42 I don't want to mix the meaning of this change. I want this change to modify the structure of the comparison without changing the meaning. I think that `localeCompare` is probably the right way to go here, but changing somewhat complicated nested ternaries is a risky enough change that I'm a little nervous changing how the comparison is done